colossalai/checkpoint_io/utils.py

```diff
@@ -1,12 +1,59 @@
+# coding=utf-8
+import importlib.util
+import sys
+from packaging import version
 from pathlib import Path
-from typing import List, Optional, Tuple
-
 import torch
+import torch.nn as nn
+import json
+from typing import List, Dict, Mapping, OrderedDict, Optional, Tuple
+from colossalai.tensor.d_tensor.d_tensor import DTensor
+import logging
 
 # ======================================
 # General helper functions
 # ======================================
 
+SAFE_WEIGHTS_NAME = "model.safetensors"
+WEIGHTS_NAME = "model.bin"
+SAFE_WEIGHTS_INDEX_NAME = "model.safetensors.index.json"
+WEIGHTS_INDEX_NAME = "model.bin.index.json"
+
+# ======================================
+# Function for check if safetensor available
+# ======================================
+
+# if sys.version_info < (3, 8):
+#     import importlib_metadata
+# else:
+#     import importlib.metadata as importlib_metadata
+
+# def is_safetensors_available():
+#     torch_available, torch_version = is_torch_available()
+#     if torch_available:
+#         if version.parse(torch_version) >= version.parse("1.10"):
+#             return importlib.util.find_spec("safetensors") is not None
+#         else:
+#             return False
+#     else:
+#         return importlib.util.find_spec("safetensors") is not None
```

Comment:
This function is still used, why comment it out?

Reply:
This function is written by me，i use the one you wrote.

Reply:
Ok, perhaps you should  delete these comments.